concrete/single_pages/dashboard/system/registration/automated_logout.php

```diff
@@ -0,0 +1,101 @@
+<?php
+defined('C5_EXECUTE') or die('Access Denied.');
+
+/* @var Concrete\Core\Form\Service\Form $form */
+/* @var Concrete\Core\Validation\CSRF\Token $token */
+/* @var Concrete\Core\Page\View\PageView $view */
+
+/* @var bool $invalidateOnIPMismatch */
+/* @var bool $invalidateOnUserAgentMismatch */
+/* @var string $trustedProxyUrl */
+/* @var string $invalidateAction */
+/* @var string $saveAction */
+?>
+<form action="<?= $saveAction ?>" method="POST">
+    <?php $token->output('save_automated_logout') ?>
+
+    <fieldset>
+        <legend><?= t('Session Security') ?></legend>
+        <div class="form-group">
+
+            <div class="help-block">
+                <?= t('These settings help prevent a user from stealing other logged in user sessions. You may want to configure %s"Trusted Proxies"%s instead', '<a href="' . $trustedProxyUrl . '">', '</a>') ?>
+            </div>
+            <div class="checkbox">
+                <label>
+                    <?= $form->checkbox('invalidateOnIPMismatch', '1', $invalidateOnIPMismatch) ?>
+                    <?= t('Log users out if their IP changes') ?>
+            </div>
+            <div class="checkbox">
+                <label>
+                    <?= $form->checkbox('invalidateOnUserAgentMismatch', '1', $invalidateOnUserAgentMismatch) ?>
+                    <?= t('Log users out if their browser\'s user agent changes') ?>
+                </label>
+            </div>
+
+        </div>
+    </fieldset>
+
+    <fieldset>
+        <legend><?= t('Invalidate Active Sessions') ?></legend>
+
+        <div class="form-group">
+            <p>
+                <?=t('Type %s in the following box to proceed.', '<code>invalidate</code>')?>
+            </p>
+            <div class="input">
+                <?= $form->text('confirmation', [
+                    'data-submit' => $invalidateAction
+                ]) ?>
+                <div class="invalid-feedback text-danger help-text">
+                    <?= t('Please type %s to proceed.', '<code>invalidate</code>') ?>
+                </div>
+            </div>
+        </div>
+
+        <div class="form-group">
+            <div class="alert alert-danger">
+                <strong><?=t('Warning:')?></strong> <?=t('This action will automatically log out all users (including yourself!)')?>
```

Comment:
I'd always use `t('Warning') . ':'` to ease translation work.

Reply:
I'll keep this in mind for the future :+1: